cli.js

```diff
@@ -22,12 +22,14 @@ const cli = meow(`
 	  --no-publish  Skips publishing
 	  --tag         Publish under a given dist-tag
 	  --no-yarn     Don't use Yarn
+	  --content     Tarball or folder to be published
```

Comment:
I don't think we should mention tarball. Even though it works, I don't see the use-case and this PR is about support publishing a specific subdirectory.

Reply:
Not sure about the name either. Shouldn't it be `contents`?

Description should be:

> Subdirectory to publish

Reply:
Good point. I updated the changes based on your comment. Could you check it again, please?